### PR TITLE
Update injection_script.js

### PR DESCRIPTION
**Pull request by EastArctica**
Example link you can test with:
https://earnshortner.com/insurance-in-the-united-states-2/?key=29d7aae60000f6b4baf76f4f568b8c60&page=1

Not sure if I did this correctly, But it works fine on that link at least(didn't test any others)

### CODE REVIEW

**Comment by FireMasterK** [comment]
Couldn't this be simplified to this?
```
domainBypass('earnshortner.com',()=>{
	ensureDomLoaded(()=>{
        if (location.search.indexOf('page=3') != -1) {
            safelyNavigate(document.getElementsByClassName('btn-link list-group-item')[0].href)
        } else if(location.search.indexOf('page') != -1) {
            safelyNavigate(location.href.slice(0, location.href.length-1) + '3')
        }
	})
})
```

**Comment by EastArctica** [comment]
Yes, It could definitely be simplified to that. There are many other better ways of doing this such as sending an XML request instead of redirecting twice. I'll update the pull request now.

File: injection_script.js [hunk 1]
```diff
@@ -667,15 +667,11 @@ domainBypass("universal-bypass.org",()=>{
...
-            safelyNavigate(location.href.slice(0, location.href.length-1) + '3')
-        }
-        if (location.search.indexOf('page=2') != -1) {
-            safelyNavigate(location.href.slice(0, location.href.length-1) + '3')
-        }
-        if (location.search.indexOf('page=3') != -1) {
-            safelyNavigate(document.getElementsByClassName('btn-link list-group-item')[0].href)
-        }
+		if (location.search.indexOf('page=1') != -1) {
+		    safelyNavigate(location.href.slice(0, location.href.length-1) + '3')
```
Comment (FireMasterK) [inline injection_script.js:671]: You made a mistake here

**Comment by Sainan** [comment]
Either the site is broken or your bypass breaks the site.

**Comment by Sainan** [comment]
Also, I love how you:
- Use spaces instead of tabs
- Placed your code in a completely random place
- Use ' instead of "

File: injection_script.js (hunk 1 above)
Comment (EastArctica) [reply injection_script.js]: Whoops, Wrote that fix on my phone and we all know how terrible of an idea that is.

**Comment by EastArctica** [comment]
Dear god I absolutely dispise people who use spaces on purpose, the reason that it used spaces was because normally when writing something like this for a site I just write it in the chrome dev console which for some god awful reason uses spaces. For the putting my code in a completely random space, Yep I had no idea where to put it at all. Using ' instead of " is from my previous lua knowledge. I really need to switch to " though because alot of languages see ' as a char instead. (fixed code it should work now, forgot to change spaces to tabs though)

**Comment by Sainan** [comment]
That's why it's always good ot know where you got the link from and not just the link itself, especially for these shorteners which act like they're a blog or something else.

**Comment by EastArctica** [comment]
Yeah, I guess I'll close this issue for now until someone finds what site this is under. (Thought I closed this pull request days ago)

**Pull request closed without merging**